core/subsurface-qt/DiveObjectHelper.cpp

```diff
@@ -55,6 +55,13 @@ static QString getPressures(struct dive *dive, int i, enum returnPressureSelecto
 	return fmt;
 }
 
+// Qt's metatype system insists on generating a default constructed object,
+// even if that makes no sense. Usage of this object *will* crash.
+DiveObjectHelper::DiveObjectHelper() :
```

Comment:
which makes me wonder if there's a way to let us know that such an object was created and who was the caller / backtrace to get us there - and to do so in a way that we can then ask the user to send us THAT log.

Reply:
The object is created when registering the metatype at application-startup.